eth/stagedsync/stage_bor_heimdall.go

```diff
@@ -252,7 +251,23 @@ func BorHeimdallForward(
 	// this becomes more prevalent this will need to be re-thought
 	var skipCount int
 
+	// allow commit every N blocks to avoid long transactions and lost progress
+	// N=1000 is not verified to be most optimal value, but works fine in unit tests
```

Comment:
it would be good to leave a comment saying this flow is mainly for running `./build/bin/integration stage_bor_heimdall` forward operation manually

Reply:
done